yjit/src/codegen.rs

```diff
@@ -4202,6 +4308,13 @@ fn gen_send_iseq(
         }
     }
 
+
+    if flags & VM_CALL_ARGS_SPLAT != 0 && flags & VM_CALL_ZSUPER != 0 {
+        // We can't handle zsuper
+        gen_counter_incr!(asm, send_iseq_zsuper);
+        return CantCompile;
+    }
```

Comment:
What is zsuper? Could we handle it before? Can we not handle it only in the context of splat calls? You could explain this in a comment above the if statement.